### PR TITLE
Feature/dialyzer on travis runs

### PR DESCRIPTION
**Pull request by hez**
- sets the plt directory to priv/plts
- settings in travis to cache the plts
- run dialyzer on travis

### CODE REVIEW

**Comment by hez** [comment]
@uesteibar looks like it took a while (since it was the first run this is expected) but future runs should be quicker as the plts will be cached and it shouldn't have to build from scratch each time.

File: .travis.yml [hunk 1]
```diff
@@ -5,10 +5,15 @@ elixir:
...
+    - priv/plts
+
 jobs:
   include:
     - stage: lint
       elixir: 1.9.1
       script:
         - mix format --check-formatted
         - mix credo
+        - MIX_ENV=dev travis_wait 20 mix dialyzer --plt
```
Comment (uesteibar) [inline .travis.yml]: Maybe this is worth running as a separate job? we could call it `dyalizer`

Comment (hez) [reply .travis.yml]: good call

**Comment by hez** [comment]
FYI second run of dialyzer took `1:30` so 👍

**Pull request merged**